src/wdio/services/SeleniumDockerService.js

```diff
@@ -17,46 +23,30 @@ export default class SeleniumDockerService {
    * @param {Object} config wdio configuration object
    * @param {Array.<Object>} capabilities list of capabilities details
    */
-  async onPrepare(config, capabilities) {
+  async onPrepare(config) {
+    // If a custom compose file is provided, dont' default instances as we don't
+    // know the service names.
```

Comment:
you can get the service names via
`docker-compose config --services`
https://docs.docker.com/compose/reference/config/